### DISCUSSION TITLE
Getting Error at Pom.xml

### DISCUSSION THREAD

**Discussion by vishwabujji** [akash-coded/spring-framework#82] [Guidelines]
![Screenshot (77)](https://user-images.githubusercontent.com/101938974/168857094-aa9c62d3-d468-439f-9259-c85c9695ac69.png)
Please Help me for This Error

**Comment by akash-coded** [comment]
@vishwabujji 
Spring Tools 4.14.1 is missing tm4e.

To install it, go to "Help → Install new software → Add URL: [https://download.eclipse.org/tm4e/snapshots/"](https://download.eclipse.org/tm4e/snapshots/%22). Click finish. Restart Eclipse.

**Comment by vishwabujji** [comment]
Thank you sir
